File: PHP/Kode_Program/Petshop.php
<?php

class Petshop {
    // Properties
    private $ID, $nama_produk, $harga_produk, $stok_produk, $foto_produk;

    // Constructor
    public function __construct(){
        $this->ID = 0;
        $this->nama_produk = "";
        $this->harga_produk = 0;
        $this->stok_produk = 0;
        $this->foto_produk = "";
    }

    // Setter
    public function Set_ID($id) { $this->ID = $id; }
    public function Set_Nama($nama) { $this->nama_produk = $nama; }
    public function Set_Harga($harga) { $this->harga_produk = max(0, (int)$harga); }
    public function Set_Stok($stok) { $this->stok_produk = max(0, (int)$stok); }
    public function Set_Foto($foto) { $this->foto_produk = $foto; }

    // Getter
    public function Get_ID() { return $this->ID; }
    public function Get_Nama() { return $this->nama_produk; }
    public function Get_Harga() { return $this->harga_produk; }
    public function Get_Stok() { return $this->stok_produk; }
    public function Get_Foto() { return $this->foto_produk; }
}
?>

File: Python/Kode_Program/Petshop.py
class Petshop:
    
    # Konstruktor untuk inisialisasi objek dengan atribut default
    def __init__(self, ID=0, nama_produk="", harga_produk=0, stok_produk=0):
        self.ID = ID
        self.Nama = nama_produk
        self.Harga = harga_produk
        self.Stok = stok_produk

    # Mengatur ID produk
    def Set_ID(self, ID):
        self.ID = ID  

    # Mengatur nama produk
    def Set_Nama(self, nama_produk):
        self.Nama = nama_produk  

    # Mengatur harga produk
    def Set_Harga(self, harga_produk):
        self.Harga = harga_produk  

    # Mengatur jumlah stok produk
    def Set_Stok(self, stok_produk):
        self.Stok = stok_produk  

    # Mengambil ID produk
    def Get_ID(self):
        return self.ID  
    
    # Mengambil nama produk
    def Get_Nama(self):
        return self.Nama  

    # Mengambil harga produk
    def Get_Harga(self):
        return self.Harga  
    
    # Mengambil jumlah stok produk
    def Get_Stok(self):
        return self.Stok  
